GameData/SPEngine/Families/F.cfg

```diff
@@ -39,14 +39,14 @@ SPEFamily:NEEDS[RealFuels]
 		mass = 9.467 // yields 2.710 for an E-1 (that's very heavy!), or 8.391 for an F-1.
 		entryCost = 300000
 		entryCost = E1-TP
-		cost = 5455 // yields 835.5 (very cheap!) for an E-1, or 4552.3 for an F-1
+		cost = 2520 // Yields 2107 for an F-1
 		toolCost = 120000 // yields 18380.2 for an E-1, or 100141.3 for an F-1
 	}
 	TechLevel
 	{
 		// F-1A = 150000, F-1
 		techRequired = orbitalRocketry1976
-		maxThrust = 10750 // yields 2251 for an upgraded E-1, or 9245.6 for an upgraded F-1
```

Comment:
Why'd you nope out these bits of comment?  They're useful documentation (the E-1 is the small end of the F-class range).

Reply:
Documentation wise they are indeed important and I'll correct this

Reply:
Still seems to be gone in latest version...?